apps/web/src/components/Feed/SuggestedProfileCard.tsx

```diff
@@ -0,0 +1,241 @@
+import { useMemo } from 'react';
+import { graphql, useFragment } from 'react-relay';
+import styled from 'styled-components';
+
+import { SuggestedProfileCardFollowFragment$key } from '~/generated/SuggestedProfileCardFollowFragment.graphql';
+import { SuggestedProfileCardFragment$key } from '~/generated/SuggestedProfileCardFragment.graphql';
+import { contexts } from '~/shared/analytics/constants';
+import { removeNullValues } from '~/shared/relay/removeNullValues';
+import { useLoggedInUserId } from '~/shared/relay/useLoggedInUserId';
+import colors from '~/shared/theme/colors';
+import unescape from '~/shared/utils/unescape';
+
+import Badge from '../Badge/Badge';
+import breakpoints from '../core/breakpoints';
+import Markdown from '../core/Markdown/Markdown';
+import { HStack, VStack } from '../core/Spacer/Stack';
+import { BaseM } from '../core/Text/Text';
+import FollowButton from '../Follow/FollowButton';
+import { ProfilePicture } from '../ProfilePicture/ProfilePicture';
+
+type Props = {
+  userRef: SuggestedProfileCardFragment$key;
+  queryRef: SuggestedProfileCardFollowFragment$key;
+  showFollowButton?: boolean;
+  onClick?: () => void;
+};
+
+export default function SuggestedProfileCard({
+  userRef,
+  queryRef,
+  onClick,
+  showFollowButton = true,
+}: Props) {
+  const user = useFragment(
+    graphql`
+      fragment SuggestedProfileCardFragment on GalleryUser {
+        id
+        username
+        badges {
+          name
+          imageURL
+          ...BadgeFragment
+        }
+        bio
+        galleries {
+          tokenPreviews {
+            large
+          }
+          hidden
+        }
+        ...ProfilePictureFragment
+        ...FollowButtonUserFragment
+      }
+    `,
+    userRef
+  );
+
+  const query = useFragment(
+    graphql`
+      fragment SuggestedProfileCardFollowFragment on Query {
+        ...FollowButtonQueryFragment
+        ...useLoggedInUserIdFragment
+      }
+    `,
+    queryRef
+  );
+
+  const loggedInUserId = useLoggedInUserId(query);
+  const isOwnProfile = loggedInUserId && loggedInUserId === user?.id;
+
+  if (!user) {
+    throw new Error('No user available to showcase SuggestedProfileCard');
+  }
+
+  const { bio } = user;
+  const unescapedBio = useMemo(() => (bio ? unescape(bio) : ''), [bio]);
```

Comment:
we shouldn't need to manually `unescape` on the frontend anymore, the backend should handle this!

Reply:
update: the backend unescaping happens in `schema.resolvers.helpers.go`: https://github.com/gallery-so/go-gallery/pull/1214/files

ask: in a follow-up PR, can you track down all the cases where the frontend calls `unescape` (screenshot), then ensure the necessary locations in the backend unescapes those fields? the end result should be that we don't call `unescape` anywhere in the frontend

<img width="530" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/ed25dc21-9b28-404f-831f-7d1b0f68e77b">